monitor.py

```diff
@@ -476,13 +476,13 @@ def get_nightly(build_log_url,build_log_response, job_platform):
         else:
             nightly = job_platform+"-latest-"+ nightly_log_match.group(2)
     else:
-        nightly_initial_log_re = re.compile('(Resolved release {}-initial to (\S+))'.format(job_platform), re.MULTILINE|re.DOTALL)
+        nightly_initial_log_re = re.compile('(Using explicitly provided pull-spec for release {}-initial \((\S+)\))'.format(job_platform), re.MULTILINE|re.DOTALL)
         nightly_initial_log_match = nightly_initial_log_re.search(build_log_response.text)
         if nightly_initial_log_match is None:
             nightly = "Unable to fetch nightly {}-initial information- No match found".format(job_platform)
         else:
             nightly = job_platform+"-initial-"+ nightly_initial_log_match.group(2)
-        nightly_latest_log_re = re.compile('(Resolved release {}-latest to (\S+))'.format(job_platform), re.MULTILINE|re.DOTALL)
+        nightly_latest_log_re = re.compile('(Using explicitly provided pull-spec for release {}-latest \((\S+)\))'.format(job_platform), re.MULTILINE|re.DOTALL)
```

Comment:
For both initial and latest image, check if this matches `re.compile('(Resolved release {}-latest to (\S+))'.format(job_platform), re.MULTILINE|re.DOTALL) `, if it doesnt matches then check `re.compile('(Using explicitly provided pull-spec for release {}-latest \((\S+)\))'.format(job_platform), re.MULTILINE|re.DOTALL)`, if this also doesnt match return Unable to fetch